### ISSUE TITLE
error with 14.9.0 under dev-mode

### ISSUE THREAD

**Issue by ZheSun88** [vaadin/skeleton-starter-flow#639]
Upgrade project to 14.9.0 failed with error when building under dev mode 
```
java.lang.NoClassDefFoundError: com/vaadin/pro/licensechecker/LicenseChecker
```

step to reproduce:
- take this branch https://github.com/vaadin/skeleton-starter-flow/pull/638
- run `mvn install -Pit`
(make sure you are using JAVA 8.)

**Comment by Artur-** [comment]
Fixed in 14.9.1?

**Comment by ZheSun88** [comment]
ah, yes.. i will close this one..
